### PR TITLE
implementing issue#15

### PR DESCRIPTION
**Pull request by kruslin2**
Please review the program code door.py implementing issue$15

### CODE REVIEW

File: door.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+shading_values = [
+	[0.6, 0.6, 0.6, 0.6],
+	[0.6, 0.6, 0.6, 0.6],
+	[1.0, 1.0, 1.0, 1.0],
+	[0.4, 0.4, 0.4, 0.4],
+	[0.8, 0.8, 0.8, 0.8],
+	[0.8, 0.8, 0.8, 0.8],
+]
+
+assert transparent == False
```
Comment (cleancoder1016) [inline door.py:40]: I find the code between lines 39 and 73 unnecessary and adds no value to the code. Please change it and also write unit test cases for the code by importing unittest module.

**Review by cleancoder1016 — CHANGES_REQUESTED** [review]
I find the code between lines 39 and 73 unnecessary and adds no value to the code. Please change it and also write unit test cases for the code by importing unittest module.

**Review by cleancoder1016 — APPROVED** [review]
I find the code between lines 39 and 73 unnecessary and adds no value to the code. Please change it and also write unit test cases for the code by importing unittest module.

**Review by kruslin2 — COMMENTED** [review]
Changed to  unittest

**Pull request merged**